tools/netsh/ebpfnetsh.rc

```diff
@@ -261,6 +261,17 @@ BEGIN
 \nUsage: %1!s!\
 \n\
 \nRemarks: Shows processes using eBPF.\
+\n"
+
+    HLP_EBPF_SHOW_HASH  "Shows hash embedded in PE file.\n"
```

Comment:
So "show hash" won't work on a `.o` file?  This seems to contradict what `ebpf_api_get_data_section()` does, which also works on a `.o` file.